add-on/panels/css/panel.css

```diff
@@ -655,6 +655,9 @@ html[dir="rtl"] .settings-menu .dropdown-menu {
   color: #000;
   font-weight: bold;
   margin: 0;
+  width: 270px;
```

Comment:
Did this 270px come from the design spec? If not, a margin/padding end might be better than a specific width as the footer width is set to 100%.